### PR TITLE
Update textfile.txt

### PR DESCRIPTION
**Pull request by Prathameshkd**
Added more information...

### CODE REVIEW

**Comment by Prathameshkd** [comment]
Please review it sooon

File: textfile.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 Hello , Iam Prathamesh Dukare
+I love programming in python
```
Comment (prathamesh-dukare) [inline textfile.txt]: "P" shoud be capital of Python

**Comment by prathamesh-dukare** [comment]
Dont mension programming language

**Comment by prathamesh-dukare** [comment]
ok, i will give you chance to merge again ,
Now do some great work

**Comment by prathamesh-dukare** [comment]
Now , iam gonna merge the PR

**Comment by prathamesh-dukare** [comment]
# Application to the Class of 2021🎓

This pull request template helps you complete an application to the **Class of 2021**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [ ] I have read the instructions on the README file before submitting my application. 
- [ ] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`) and markdown file.
- [ ] I have submitted a [swag shipping form](https://airtable.com/shrM5IigBuRFaj33H).
- [ ] I have used the Markdown file template to add my information to the Year Book.
- [ ] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [ ] I understand I should not tag or add a reviewer to this Pull Request.
- [ ] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20210605T160000Z%2F20210605T173000Z&text=GitHub%20Graduation%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

**Comment by prathamesh-dukare** [comment]
# Application to the Class of 2021🎓

This pull request template helps you complete an application to the **Class of 2021**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [ x] I have read the instructions on the README file before submitting my application. 
- [ ] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`) and markdown file.
- [ ] I have submitted a [swag shipping form](https://airtable.com/shrM5IigBuRFaj33H).
- [ ] I have used the Markdown file template to add my information to the Year Book.
- [ ] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [ ] I understand I should not tag or add a reviewer to this Pull Request.
- [ ] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20210605T160000Z%2F20210605T173000Z&text=GitHub%20Graduation%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

**Comment by prathamesh-dukare** [comment]
# Application to the Class of 2021🎓

This pull request template helps you complete an application to the **Class of 2021**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [ X] I have read the instructions on the README file before submitting my application. 
- [ ] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`) and markdown file.
- [ ] I have submitted a [swag shipping form](https://airtable.com/shrM5IigBuRFaj33H).
- [ ] I have used the Markdown file template to add my information to the Year Book.
- [ ] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [ ] I understand I should not tag or add a reviewer to this Pull Request.
- [ ] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20210605T160000Z%2F20210605T173000Z&text=GitHub%20Graduation%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

**Pull request merged**